### DISCUSSION TITLE
How likely to have two faulty Faikins??

### DISCUSSION THREAD

**Discussion by PianSom** [revk/xESP32-Faikout#294] [General]
A couple of months ago I bought a Faikin for an existing unit, fitted it and it worked just fine. Love it.

I recently got a new double header unit fitted, and bought 2 EKRS21s (which are needed for the current model) and 2 more Faikins. The two indoor units are identical. I fitted up one unit - worked just fine. Love it too. 

But the third one showed no signs of life :( So I returned the Faikin to Amazon, and was told by A+A that it must've been faulty as that happens occasionally. I bought a new one. I just installed this one - dead too!

How likely is it to have come across two faulty Faikins? It seems pretty unlikely to me. I have used my circuit tester to check the connection between the EKRS21 pins and the Faikin pins (connected, like the other, with Dupont cables) - all good. But if I put 5V across the Faikin 5V + Earth - nothing.

My question is - should I get some sort of life from a Faikin if I put 5V across it? Or have I burnt it out by trying? Or have I just been colossally unlucky, and need to return this one too?

**Comment by revk** [comment]
Swap around known working unit each way to confirm. Yes, unlikely to be two faulty units!

Faikin is not powered by the 5V line (pin 1), it is powered by the 12V line (pin 4), which can be 4V to 35V.

A "burn out" is hard.

**Comment by PianSom** [comment]
Thanks! 

Putting 5V across pins 1 and 4 - the Faikin springs to life! Time to re-check the Duponts and EKRS21

EDIT - cable checks out. Maybe it's the board in the unit. Looks like I will have to open up the working one after all. <sigh>

**Reply by PianSom** [reply]
Think it was a bit of crud in the EKRS21 - all working properly now.

Thanks for the extremely prompt and helpful response!

**Reply by revk** [reply]
No way to tell Amazon the part was OK I expect. I'll see what they do. But thanks for letting me know.

**Reply by PianSom** [reply]
Luckily my Amazon conversation with A+A was still openable. I have messaged them to tell them to check the return and that it is probably re-sellable

If they can't resell because I took off the excess pcb then I am happy to take the hit

**Reply by revk** [reply]
Thanks, they did that once before and made a "used, as new" product code, so cool.
